client/components/dataviews/style.scss

```diff
@@ -228,10 +228,6 @@
 // Follow-up issue: https://github.com/Automattic/dotcom-forge/issues/9554
 body.is-section-plugins .plugin-management-wrapper {
     --wp-components-color-accent: var(--color-accent);
-
-	.dataviews-loading {
```

Comment:
With the display flex, the spinner loads in the middle of the screen, which is even better.